pkg/driver/node.go

```diff
@@ -261,10 +259,7 @@ func (d *nodeService) NodeExpandVolume(ctx context.Context, req *csi.NodeExpandV
 	}
 
 	// TODO: refactor Mounter to expose a mount.SafeFormatAndMount object
-	r := resizefs.NewResizeFs(&mount.SafeFormatAndMount{
-		Interface: mount.New(""),
-		Exec:      exec.New(),
-	})
+	r := mountutils.NewResizeFs(d.mounter.(*NodeMounter).Exec)
```

Comment:
Why this change? may be adding some details in the PR description may help the reviewer

Reply:
PR Description updated